python/ray/tune/tune.py

```diff
@@ -143,6 +146,14 @@ def run(run_or_experiment,
         loggers (list): List of logger creators to be used with
             each Trial. If None, defaults to ray.tune.logger.DEFAULT_LOGGERS.
             See `ray/tune/logger.py`.
+        log_to_file (bool|str|Sequence): Log stdout and stderr to files in
+            Tune's trial directories. If this is `False` (default), no files
+            are written. If `true`, outputs are written to `trialdir/stdout`
+            and `trialdir/stderr`, respectively. If this is a single string,
+            this is interpreted as a file relative to the trialdir, to which
+            both streams are written. If this is a Sequence, it has to have
+            length 2 and the elements indicate the files to which stdout and
+            stderr are written, respectively.
```

Comment:
Tuple[str]?

Reply:
I think we'd like to support lists, too, or at least offer the flexibility to pass both. A python Sequence supports `__len__` and `__getitem__`, so I think it's the best abstraction to use here.
However, I updated the docstring to includ Tuple as the preferred example.